src/util/mapbox.js

```diff
@@ -12,7 +12,7 @@ type UrlObject = {|
     params: Array<string>
 |};
 
-function makeAPIURL(urlObject: UrlObject, accessToken): string {
+function makeAPIURL(urlObject: UrlObject, accessToken?: ?string): string {
```

Comment:
It needs (and is able) to accept both `null` and `undefined`. It would be clearer to write this as `accessToken: string | null | void` though -- I'll switch.